src/reducers/dataReducer.js

```diff
@@ -49,7 +48,7 @@ export function GRIDDLE_INITIALIZED(initialState) {
  * @param {Object} action - The action object to work with
 */
 export function GRIDDLE_LOADED_DATA(state, action) {
-  const transformedData = transformData(action.data);
+  const transformedData = transformData(action.data, state.get('renderProperties').toJSON());
```

Comment:
We _can_, but I think it'd going to lead to something that's a bit of a bummer during `GRIDDLE_INITIALIZE`, where the initial conversion into Immutable happens.

The alternative to `toJSON()` here would be converting the `renderProperties` in `GRIDDLE_INITIALIZE` to a `Map` before the rest of the initial state and then executing `set('renderProperties', renderPropertiesMap)`.

Both options seem pretty lame, heh.